### PR TITLE
add transaction success xml page

### PR DESCRIPTION
**Pull request by kimhajiny**

### CODE REVIEW

File: Test2/app/src/main/res/layout/activity_transaction_success.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        android:layout_marginTop="170dp"
+        android:contentDescription="image"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@drawable/t_pay_image"
+        tools:ignore="ImageContrastCheck" />
+
+    <TextView
+        android:id="@+id/textView"
```
Comment (Chaewon382) [inline Test2/app/src/main/res/layout/activity_transaction_success.xml:28]: Same problem here.

File: Test2/app/src/main/res/layout/activity_transaction_success.xml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    android:layout_height="match_parent"
+    android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingBottom="@dimen/activity_vertical_margin"
+    tools:context=".TransactionSuccess">
+
+
+    <ImageView
+        android:id="@+id/imageView"
```
Comment (Chaewon382) [inline Test2/app/src/main/res/layout/activity_transaction_success.xml:16]: ID should be descriptive enough to indicate what this component is for.

**Review by Chaewon382 — COMMENTED** [review]
Left an inline comment. I suggest you make the id's of your xml components more descriptive. Other that that, great work :)

**Review by taehee5525 — COMMENTED** [review]
Everything looks good to me except that one thing Chaewon mentioned above.

**Pull request merged**